style.css

```diff
@@ -1040,3 +1040,2010 @@ body {
     color: black;
   }
 }
+
+@media screen and (min-width: 620px) {
```

Comment:
- [ ] As you can clearly see in the screenshot below you have another scrollbar added in your Portfolio the thing that deforms your project design, Kindly work on refactoring that by adding `overflow: hidden;` in your CSS, you can read more about that [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)


![overflow](https://user-images.githubusercontent.com/80895497/150239240-933e2fe3-1931-452a-8655-7e5fe1701099.PNG)

Reply:
Thank you

Reply:
> * [ ]  As you can clearly see in the screenshot below you have another scrollbar added in your Portfolio the thing that deforms your project design, Kindly work on refactoring that by adding `overflow: hidden;` in your CSS, you can read more about that [here](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow)
> 
> ![overflow](https://user-images.githubusercontent.com/80895497/150239240-933e2fe3-1931-452a-8655-7e5fe1701099.PNG)

Hello Hamza, about this, I already used "overflow: hidden" right at the beginning and it was supposed to run through the entire project.

Reply:
Hey @carlylechia try to use `overflow-y: hidden` or `overflow-x: hidden`